views/pulsar/components/tab.html.twig

```diff
@@ -7,26 +7,39 @@
 
 {% set _hasSettings = false %}
 {% set _hasSidebar = false %}
+{% set _actions_left = '' %}
+{% set _actions_right = '' %}
 
-{% set _settings = block('tab_settings') %}
-{% if _settings is not empty %}
-    {% set _hasSettings = true %}
+{% if block('tab_settings') is defined %}
+    {% set _settings = block('tab_settings') %}
+    {% if _settings is not empty %}
+        {% set _hasSettings = true %}
+    {% endif %}
+{% endif %}
+
+{% if block('tab_sidebar') is defined %}
+    {% set _sidebar = block('tab_sidebar') %}
+    {% if _sidebar is not empty %}
+        {% set _hasSidebar = true %}
+    {% endif %}
 {% endif %}
 
-{% set _sidebar = block('tab_sidebar') %}
-{% if _sidebar is not empty %}
-    {% set _hasSidebar = true %}
+{% if block('tab_js') is defined %}
+    {% set _tab_javascript = block('tab_js') %}
+    {% if _tab_javascript is not empty %}
+        {% block tab_javascript %}
+            {{ _tab_javascript }}
+        {% endblock %}
+    {% endif %}
 {% endif %}
 
-{% set _tab_javascript = block('tab_js') %}
-{% if _tab_javascript is not empty %}
-    {% block tab_javascript %}
-        {{ _tab_javascript }}
-    {% endblock %}
+{% if block('actions_left') is defined %}
+    {% set _actions_left = block('actions_left') %}
 {% endif %}
 
-{% set _actions_left = block('actions_left') %}
-{% set _actions_right = block('actions_right') %}
+{% if block('actions_right') is defined %}
+    {% set _actions_right = block('actions_right') %}
+{% endif %}
```

Comment:
We don't need to put these into variables. We can just check if the block is defined below and then output the `block('actions_right')` directly.